src/FSharpVSPowerTools/Commands/HighlightUsageTaggerProvider.cs

```diff
@@ -18,6 +18,36 @@ namespace FSharpVSPowerTools
     [TagType(typeof(HighlightUsageTag))]
     public class HighlightUsageTaggerProvider : IViewTaggerProvider
     {
+        //[Import]
+        //internal VSLanguageService fsharpVsLanguageService = null;
+
+        //[Import]
+        //internal ITextDocumentFactoryService textDocumentFactoryService = null;
+
+        //[Import(typeof(SVsServiceProvider))]
+        //internal IServiceProvider serviceProvider = null;
+
+        //[Import]
+        //internal ProjectFactory projectFactory = null;
+
+        //private readonly IServiceProvider _serviceProvider;
+        //private readonly ITextDocumentFactoryService _textDocumentFactoryService;
+        //private readonly ProjectFactory _projectFactory;
+        //private readonly VSLanguageService _fsharpVsLanguageService;
+
+        //[ImportingConstructor]
+        //public HighlightUsageTaggerProvider(
+        //    IServiceProvider serviceProvider,
```

Comment:
If we add `[Import(typeof(SVsServiceProvider))]`, we can go with `ImportingConstructor` refactoring.